scripts/codegen/codegen.ts

```diff
@@ -0,0 +1,22 @@
+import { CodeGenCommand } from '../../packages/codegen/src';
+import { yargsBase } from './command';
+
+const cmd = new CodeGenCommand();
+
+const parser = yargsBase().options({
+    package: { type: 'string', demand: true,  describe: "Js sdk package (stable, beta, etc.)" },
```

Comment:
These files (in the top level `scripts` dir) is not linted or formatted. I think the easiest fix is to move it into the codegen package so it will be linted by its setup